frontend/src/components/NotificationsCentral/style.scss

```diff
@@ -0,0 +1,42 @@
+@use '../../theme/variables.scss' as vars;
+
+.#{vars.$bcgov-prefix}--row{
+  margin: 0rem;
+}
+
+.btn-viewed {
+  padding-right: 3.75rem;
+}
+
+.btn-all-notifications {
+  padding-right: 3.31rem;
+}
+
+.empty-section-container {
+  padding: 1.5rem 1rem;
+}
+
+p.subtitle-section {
+  font-size: 0.875rem;
+  font-weight: normal;
+}
+
+hr {
+  border-top: 1px solid #DFDFE1;
```

Comment:
Use a defined css var to apply the color, I think `--bcgov-border-subtle-00` is the correct one for this case, but please confirm on figma.